### DISCUSSION TITLE
Support JSON5 and JavaScript configs for Shareable Config Presets?

### DISCUSSION THREAD

**Discussion by karlhorky** [renovatebot/renovate#14518] [Request Help] [answered]
### What would you like Renovate to be able to do?

Hi there, first of all, thanks for your continued work on Renovate, it's amazing! 🙌

It would be great if [Shareable Config Presets](https://docs.renovatebot.com/config-presets/) (first implemented in https://github.com/renovatebot/renovate/issues/647 by @rarkins ) would be able to also support JSON5 and JavaScript configuration files, to allow comments and other constructs in the configuration files.

### If you have any ideas on how this should be implemented, please tell us here.

Since there is an existing configuration parsing engine, maybe use this also for the Shareable Config Presets?

### Is this a feature you are interested in implementing yourself?

Maybe

**Comment by viceice** [comment]
We can simply switch to `JSON5` parser to resolve this.

**Comment by viceice** [comment]
I think there is already a feature request.

**Comment by karlhorky** [comment, marked as answer]
Ah, sorry, that's over here: #7674

Closing...
